src/brackets.js

```diff
@@ -205,6 +206,7 @@ define(function (require, exports, module) {
             ScrollTrackMarkers      : require("search/ScrollTrackMarkers"),
             UpdateNotification      : require("utils/UpdateNotification"),
             WorkingSetView          : WorkingSetView,
+            XMLUtils                : XMLUtils,
```

Comment:
Ah ok, didn't know that.
I only wanted him do this as I saw `JSUtils`, another non-UI module, referenced in the same object.